__tests__/app.test.js

```diff
@@ -115,6 +115,74 @@ describe('/api/reviews/:review_id', () => {
                 });
         });
     });
+    describe('PATCH', () => {
+        test('should update the votes of a review (adding)', () => {
+            const updateVotes = {inc_votes: 50}
+            return request(app)
+            .patch("/api/reviews/1")
+            .send(updateVotes)
+            .expect(200)
+            .then(({body}) => {
+                expect(body.review).toEqual({"category": "euro game",
+                   "created_at": "2021-01-18T10:00:20.514Z",
+                   "designer": "Uwe Rosenberg",
+                   "owner": "mallionaire",
+                   "review_body": "Farmyard fun!",
+                   "review_id": 1,
+                   "review_img_url": "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+                   "title": "Agricola",
+                   "votes": 51,})
+            })
+        });
+        test('should update the votes of a review (subracting)', () => {
+            const updateVotes = {inc_votes: -50}
+            return request(app)
+            .patch("/api/reviews/1")
+            .send(updateVotes)
+            .expect(200)
+            .then(({body}) => {
+                expect(body.review).toEqual({"category": "euro game",
+                   "created_at": "2021-01-18T10:00:20.514Z",
+                   "designer": "Uwe Rosenberg",
+                   "owner": "mallionaire",
+                   "review_body": "Farmyard fun!",
+                   "review_id": 1,
+                   "review_img_url": "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+                   "title": "Agricola",
+                   "votes": -49,})
+            })
+        });
+        test("should return a 404 if a request is made to a review which doesn't exist", () => {
+            const updateVotes = {inc_votes: 50}
+            return request(app)
+            .patch("/api/reviews/1000000")
+            .send(updateVotes)
+            .expect(404)
+            .then(({body}) => {
+                expect(body.msg).toEqual('review does not exist')
+            })
+        });
+        test("should return a 400 if the id is not a number", () => {
+            const updateVotes = {inc_votes: 50}
+            return request(app)
+            .patch("/api/reviews/a")
+            .send(updateVotes)
+            .expect(400)
+            .then(({body}) => {
+                expect(body.msg).toEqual('Bad request')
+            })
+        });
+        test("should return a 400 if inc_votes is not a number", () => {
+            const updateVotes = {inc_votes: 'a'}
+            return request(app)
+            .patch("/api/reviews/1")
+            .send(updateVotes)
+            .expect(400)
+            .then(({body}) => {
+                expect(body.msg).toEqual('Bad request')
+            })
+        });
+    });
```

Comment:
Error tests looking good! Should be using `toBe` instead of `toEqual` for primitive comparisons though.
Just one more test to consider here I think
- What if `inc_votes` isn't even there